src/slider/slider.stories.ts

```diff
@@ -16,7 +16,7 @@ storiesOf("Components|Slider", module).addDecorator(moduleMetadata({
 }))
 .addDecorator(withKnobs)
 .add("Basic", () => ({
-	template: `<ibm-slider [disabled]="disabled"></ibm-slider>`,
+	template: `<ibm-slider [disabled]="disabled" aria-label="slider"></ibm-slider>`,
```

Comment:
The point of `aria-label` if I'm not wrong is to represent what the slider is sliding through, not that it's a slider. It being a slider will be determined using `role="slider"`.

So for instance your slider's possible aria-label values would be:
`budget`, `yearly income`, `number of servers to spin up` etc.